test/system/252-quadlet.bats

```diff
@@ -558,4 +558,28 @@ EOF
     remove_secret $SECRET_NAME
 }
 
+@test "quadlet - volume path using specifier" {
+    local tmp_path=$(mktemp -d --tmpdir quadlet.volume.XXXXXX)
+    local tmp_dir=$(basename "$tmp_path")
+    local file_name=hello.txt
+    local file_content="hello"
```

Comment:
Why not use `random_string` here too? That eliminates a (small, unlikely) number of false negatives.
```bash
    local file_name="f$(random_string 10).txt"
    local file_content="data_$(random_string 15)"
```

Reply:
Sure, great idea. I'll update it as part of whatever we decide on the previous comment

Reply:
Done